### PR TITLE
Fix documentation.

### PR DESCRIPTION
**Pull request by wammar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,24 +54,24 @@ S2ORC is now available through the [Semantic Scholar Public API](https://www.sem
...
 **Twitter** [@kylelostat](https://twitter.com/kylelostat), [@lucyluwang](https://twitter.com/lucyluwang)
 
 **IRC:** `#s2orc` at `irc.oftc.net`
 
 **Give us Feedback:**  Totally optional, but we'd love to hear how you're using this dataset & any feedback for improving it.  Send us an email or leave a Github Issue. 
 
-**Report issues:** Use [GitHub Issues](https://github.com/allenai/s2orc/issues) to report bugs or issues!  We'll try to fix it for the next release.
+**Report issues:** 
+* For questions, feature requests, bug reports, please search existing issues on [the s2-folks Github repo](https://github.com/allenai/s2-folks/issues?q=is%3Aissue) before creating [a new issue](https://github.com/allenai/s2-folks/issues/new). 
+* Use [GitHub Issues](https://github.com/allenai/s2orc/issues) to report bugs or issues in the original S2ORC dataset!  We'll try to fix it for the next release.
```
Comment (wammar) [inline README.md:74]: It's unclear to me what kind of bugs/issues you'd like people to report in https://github.com/allenai/s2orc/issues. @kyleclo @lucylw

File: README.md [hunk 2]
```diff
@@ -76,16 +76,14 @@ The best way to contact us is through email.  Don't hesitate to reach out about
...
 
-- S2AG is everything that is covered in the literature graph, including Nodes (i.e. papers, authors) and Edges (i.e. citations, authorship). A `paper` in S2AG is represented by a bundle of Metadata, such as the Title, Authors, Year, Venue, Abstract, etc.
-
-- S2ORC is everything that is machine-readable **full text** of the paper, which we derive using models run on the paper's PDF. 
-
-If you're unsure what to use, please email us and we'd be happy to discuss your project with you.
+- S2ORC is everything that is machine-readable **full text** of the paper, which we derive using models run on the paper's PDF. The original S2ORC dataset files are no longer available for download. They were refactored into multiple datasets available through [the Semantic Scholar APIs](https://api.semanticscholar.org/) (See detailed documentation [here](https://api.semanticscholar.org/api-docs/datasets)).  
 
+If you're unsure what to use or cite, please email us and we'd be happy to discuss your project with you.
```
Comment (lucylw) [inline README.md:86]: should we specify who to email?

File: README.md (hunk 1 above)
Comment (lucylw) [reply README.md:74]: We're not doing future releases, so maybe let's remove this last line

**Review by lucylw — APPROVED** [review]
looks good; i'm gonna merge in

**Pull request merged**